packages/rps/src/redux/login/saga.ts

```diff
@@ -66,5 +67,6 @@ export default function* loginRootSaga() {
 }
 
 function getLibraryAddress() {
-  return process.env.RPS_CONTRACT_ADDRESS;
+  return '0x0'; // TODO fix this
+  // return process.env.RPS_CONTRACT_ADDRESS;
```

Comment:
I couldn't get this to work :-(

Reply:
I think you need this: https://github.com/statechannels/monorepo/blob/master/packages/wallet/config/env.js#L47-L53